pkg/tfbridge/detailed_diff.go

```diff
@@ -181,7 +181,7 @@ func makeBaseDiff(old, new resource.PropertyValue) baseDiff {
 		return deleteDiff
 	}
 
-	if new.IsComputed() {
+	if new.IsComputed() || new.IsOutput() {
```

Comment:
This line is not right. Output may be known. I think that you might get away with this (since Bridge claims it does not support first-class outputs).

```
contract.Assertf(!val.IsOutput())
```

Reply:
It is not right but this is how we handle outputs in schema.go - I've reverted the change however